include/calculate/c-interface.h

```diff
@@ -1,8 +1,17 @@
 #ifndef __CALCULATE_C_INTERFACE_H__
 #define __CALCULATE_C_INTERFACE_H__
 
+#ifdef _MSC_VER
+#define EXPORT_GLOBAL __declspec(dllimport)
```

Comment:
The `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` [only works for methods](https://cmake.org/cmake/help/v3.4/prop_tgt/WINDOWS_EXPORT_ALL_SYMBOLS.html), shared libraries under **Windows** were failing at linking because of this. The only solution is to add the `__declspec(dllimport)` directive when using **MSVC** compiler.